plugins/spigot/src/main/java/com/managemc/spigot/config/SpigotPluginConfigLocal.java

```diff
@@ -86,6 +86,9 @@ public class SpigotPluginConfigLocal implements SpigotPluginConfig {
   private WatchlistService watchlistService;
   @Getter
   @Setter
+  private ReportsService reportsService;
```

Comment:
Add the newly created service (scroll down) to the `SpigotPluginConfig` implementation.

This class, by the way, gets passed around as the sole argument to constructors pretty much everywhere. We tried passing individual parameters at first, but it stops working well when you have 7+ of them.